SillyBusinessInc/SillyBusinessGame
# feat/#124-healthbar-rework

## Purpose of this PR:
healthbar now works in combination with the maxhealth in playerstatistic

## How to test:
scenes with the playerprefab and healthbar prefab. you can change the maxHealth in Player to see if the amount of hearts will be the same as the amount of hp (1hp is a half heart, 2hp is 1 heart)

### links: 
[Ticket](https://github.com/orgs/SillyBusinessInc/projects/1/views/1?pane=issue&itemId=87947142&issue=SillyBusinessInc%7CSillyBusinessGame%7C124)
